### ISSUE TITLE
Upgrade event-streaming-processor to create transactions from parent spans - II

### ISSUE THREAD

**Issue by vgenev** [mojaloop/project#1039]
### *Follow up of #1013*
As Hub TechOps
I want to _be able to visualize end-to-end times of transactions (not transfers, essentially)_
so that _it gives an overview of the system performance in general_ for specific messages

This specific story is to upgrade the event-streaming-processor so that APM agent creates transactions out of root spans and close them whenever the trace ends. The association between transaction id and transferId should be persisted in memory or some storage.

**Tasks**:
- [x] Refactor the event streaming processor to remove the sleep while recreating the spans in correct order
- [x] Add starting criteria
- [x] Add handling of stale traces in the cache
- [x] Update helm charts
- [x] Find a way to test that spans are correctly linked via Postman tests
*created new story for that task* - https://github.com/mojaloop/project/issues/1063
- [x] Update event framework diagrams 

**Acceptance Criteria**:
- [x] Designs are up-to date
- [x] APM Dashboard visualize transactions as one trace
- [x] Changes made to config (default.json) are broadcast to team and follow-up tasks added to update helm charts and other deployment config.
- [x] End-to-end times for transactions can be made available for visualization

## **Pull Requests**:
- [x] https://github.com/mojaloop/event-stream-processor/pull/6
- [x] https://github.com/mojaloop/event-stream-processor/pull/11
- [ ] https://github.com/mojaloop/documentation/pull/134

## **Follow-up**:
- N/A

**Dependencies**:
- #1013 

## **Accountability**:
- Owner: TBC
- QA/Review: TBC

**Comment by kjw000** [comment]
are we going to reassign this to 8.5?

**Comment by elnyry** [comment]
Yes Kim, possibly. Will reassess after #1000 and #1034

**Comment by kjw000** [comment]
thanks @elnyry
